third_party/nvfuser/csrc/utils.h

```diff
@@ -118,6 +120,32 @@ TORCH_CUDA_CU_API bool isOptionEnabled(EnableOption option);
 TORCH_CUDA_CU_API const std::vector<std::string>& getEnableOptionArguments(
     EnableOption option);
 
+TORCH_CUDA_CU_API bool isDebugDumpEnabled(DebugDumpOption option);
```

Comment:
Is this a duplicate of another declaration at line 77?

Looks like the below `getDebugDumpArguments` is also a duplicate.